guides/source/active_record_postgresql.md

```diff
@@ -290,23 +290,48 @@ irb> article.status_published?
 => false
 ```
 
-To add a new value before/after existing one you should use [ALTER TYPE](https://www.postgresql.org/docs/current/static/sql-altertype.html):
+To rename the enum you can use `rename_enum` along with updating any model
+usage:
 
 ```ruby
-# db/migrate/20150720144913_add_new_state_to_articles.rb
-# NOTE: ALTER TYPE ... ADD VALUE cannot be executed inside of a transaction block so here we are using disable_ddl_transaction!
-disable_ddl_transaction!
+# db/migrate/20150718144917_rename_article_status.rb
+def change
+  rename_enum :article_status, to: article_state
+end
+```
+
+```ruby
+# app/models/article.rb
+class Article < ApplicationRecord
+  enum state: {
+    draft: "draft", published: "published"
+  }, _prefix: true
+end
+```
+
+To add a new value you can use `add_enum_value`:
 
+```ruby
+# db/migrate/20150720144913_add_new_state_to_articles.rb
 def up
-  execute <<-SQL
-    ALTER TYPE article_status ADD VALUE IF NOT EXISTS 'archived' AFTER 'published';
-  SQL
+  add_enum_value :article_status, "archived", # will be at the end after published
+  add_enum_value :article_status, "in review", before: "published"
+  add_enum_value :article_status, "approved", after: "in review"
```

Comment:
Do we want to change, as `article_status` enum rename to `article_state` in the above example?
```suggestion
  add_enum_value :article_state, "archived", # will be at the end after published
  add_enum_value :article_state, "in review", before: "published"
  add_enum_value :article_state, "approved", after: "in review"
```